shared/src/packages/articles/libs/enums/article-validation-message.enum.ts

```diff
@@ -0,0 +1,14 @@
+const ArticleValidationMessage = {
+  ARTICLE_TITLE_REQUIRE: 'Title is required',
+  ARTICLE_TITLE_MIN_LENGTH: 'Title must contain at least 3 characters',
+  ARTICLE_TITLE_MAX_LENGTH: 'Title should contain no more than 30 characters',
```

Comment:
Use `ArticleValidationRule` also for validation message

```suggestion
  ARTICLE_TITLE_MAX_LENGTH: `Title should contain no more than ${ArticleValidationRule.TITLE_MAX_LENGTH} characters`,
```

Reply:
Done - [76a0c38](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/59/commits/76a0c382b4836eca13173448752ca51314fe61f3)